core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -697,6 +709,37 @@ func (o *evmTxStore) loadEthTxesAttemptsReceipts(ctx context.Context, etxs []*Tx
 	return nil
 }
 
+// loadEthTxesAttemptsWithPartialReceipts loads ethTxes with partial attempts values for optimization
```

Comment:
nit: the attempts are still normal and it's just the receipts that are partial. So maybe something like this is better:
```suggestion
// loadEthTxesAttemptsWithPartialReceipts loads ethTxes with attempts and partial receipts
// receipts contain BlockHash, BlockNum, TxHash, and TransactionIndex
```

Reply:
ah thanks, typo by mistake